doc/source/user/building.rst

```diff
@@ -24,8 +24,9 @@ Building NumPy requires the following software installed:
 2) Compilers
 
    To build any extension modules for Python, you'll need a C compiler.
-   Various NumPy modules use FORTRAN 77 libraries, so you'll also need a
-   FORTRAN 77 compiler installed.
+   Various NumPy modules use FORTRAN 77 libraries, so you'll optionally need 
```

Comment:
The "various NumPy modules" isn't really true right? It's only `numpy.f2py` tests; there is no other F77 code in NumPy itself.

How about instead you say: _A Fortran compiler is not necessary to build NumPy. Only the `numpy.f2py` tests need a Fortran compiler; for more information see :ref:`choosing-the-fortran-compiler`_.

Reply:
Yes, this sounds better! I had thought about it but was unsure about introducing tests before the _Testing_ section.

Reply:
I'd incline to removing the Fortran reference and combining the two paragraphs.

Reply:
@rgommers Suggestion is also a good one.

Reply:
Thanks for your suggestion @charris! I'll make the changes soon.